tests/bgv_to_openfhe/cast.mlir

```diff
@@ -0,0 +1,42 @@
+// RUN: heir-opt --bgv-to-openfhe %s | FileCheck %s
```

Comment:
very small suggestion if you have time - to add a quick test on the output of one of these tests (extsi + encode) to the openfhe emitter test: https://github.com/google/heir/blob/6ba86a86beee53c25f12a70b73a7511100567263/tests/openfhe/ops.mlir

Reply:
Sure, I pushed a commit for it. I can squash them if it looks good.